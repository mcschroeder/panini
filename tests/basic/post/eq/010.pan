eq : (a:ℤ) → (b:ℤ) → { c:𝔹 | c = true ⟺ a = b }

f10 : {n:int|true} -> {v:bool|?}
f10 = \x:int. eq x 1

f11 : {n:int|true} -> {v:bool|?}
f11 = \x:int. eq 1 x

