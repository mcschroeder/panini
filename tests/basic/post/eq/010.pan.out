f10 : {n:ℤ | true} → {v:𝔹 | true}
f11 : {n:ℤ | true} → {v:𝔹 | true}